android/fastlane/Fastfile

```diff
@@ -73,6 +73,11 @@ platform :android do
     supply
   end
 
+  desc "Download universal APK file from Google Play"
+  lane :download_apk do
+    download_universal_apk_from_google_play
+  end
+
```

Comment:
We need at least `fastlane`, [version 2.214.2](https://github.com/fastlane/fastlane/releases/tag/2.214.0). We are actually locked at [2.212.2](https://github.com/pagopa/io-app/blob/5f64dcaafb97be1d7451405563c3e2bfd4aa13b8/Gemfile.lock#L120)